errors.go

```diff
@@ -5,6 +5,9 @@ import (
 	"strings"
 )
 
+// Ensure ErrGuac implements error
+var _ error = (*ErrGuac)(nil)
+
```

Comment:
[NewError](https://github.com/wwt/guac/pull/8/files#diff-76a7e0e299c7417bae32d3098e71370980157fe29e68a366671491d147d40572L93) below returns ErrGuac as an error, so this line seems redundant?

Reply:
Yes, it was a leftover from a previous try. I am going to delete it in the next commit!